### ISSUE TITLE
3 start-up questions for "make hgdp_tgp_sgdp_high_cov_ancients_chr20.dated.trees"

### ISSUE THREAD

**Issue by jielab** [awohns/tsdate_paper#35]
Hi, there:

I am following this instruction to run your program https://github.com/awohns/unified_genealogy_paper/blob/master/all-data/README.md

1. For "Note: The Afanasievo datasets must be separately downloaded from [here]", This is just for my information. I do NOT need to actually click that link to download files there, correct? It seems that the code has scripts to download those files.

2. For "To build the dated tree sequence of all datasets, run: make hgdp_tgp_sgdp_high_cov_ancients_chr20.dated.trees", does "all datasets" here mean all 22 autosomes of the 1000 genomes data, or does it mean all 22 autosomes of the 1000 genomes data plus the ancient genome data?

3. once I finished running the script "make hgdp_tgp_sgdp_high_cov_ancients_chr20.dated.trees", what else I need to do in order to generate a earth figure as shown in your paper?

Thank you very much!

JH

**Comment by hyanwong** [comment]
Hi @jielab - not quite on-topic, but I have put instructions for making plots similar to those in our paper using the downloadable datasets on Zenodo:

https://github.com/awohns/unified_genealogy_paper/blob/master/README.md#reproducing-plots-from-downloaded-data

**Comment by hyanwong** [comment]
> Hi, there:
> 
> I am following this instruction to run your program https://github.com/awohns/unified_genealogy_paper/blob/master/all-data/README.md
> 
> 1. For "Note: The Afanasievo datasets must be separately downloaded from [here]", This is just for my information. I do NOT need to actually click that link to download files there, correct? It seems that the code has scripts to download those files.

Yes, sorry about that. You are right. I have pushed a PR to change that: https://github.com/awohns/unified_genealogy_paper/pull/38

> 2. For "To build the dated tree sequence of all datasets, run: make hgdp_tgp_sgdp_high_cov_ancients_chr20.dated.trees", does "all datasets" here mean all 22 autosomes of the 1000 genomes data, or does it mean all 22 autosomes of the 1000 genomes data plus the ancient genome data?

"all datasets" means hgdp, tmp, sgdp, and high coverage ancients. Bt it will only do so for chromosome 20.

> 3. once I finished running the script "make hgdp_tgp_sgdp_high_cov_ancients_chr20.dated.trees", what else I need to do in order to generate a earth figure as shown in your paper?

There are instructions at https://github.com/awohns/unified_genealogy_paper#plotting-figures - let me know if this helps.
